hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/converter/JsonToAvroSchemaConverterConfig.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.utilities.schema.converter;
+
+import org.apache.hudi.common.config.ConfigProperty;
+
+/**
+ * Configs for `JsonToAvroSchemaConverter`
+ */
+public class JsonToAvroSchemaConverterConfig {
+  public static final ConfigProperty<Boolean> CONVERT_DEFAULT_VALUE_TYPE = ConfigProperty
+      .key("hoodie.streamer.schemaconverter.json_to_avro.convert_default_value_type")
+      .defaultValue(true)
+      .sinceVersion("0.14.1")
```

Comment:
These are new configs.  Let's mark the since version of both configs as `1.1.0`.